lib/api_client.ex

```diff
@@ -32,6 +19,10 @@ defmodule GlassFactoryApi.ApiClient do
   end
 
   defp url(resource) do
-    "https://#{@access_info[:subdomain]}.glassfactory.io/api/public/v1/#{resource}"
+    "#{api_domain()}/api/public/v1/#{resource}"
```

Comment:
Don't we need to also concat `@access_info[:subdomain]` here also? 👀

(If we go with the suggestion in [this comment]() then this very comment does not make sense 😄).